custom-elements/perform-microtask-checkpoint-before-construction.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Custom Elements: create an element for a token must perform a microtask checkpoint</title>
+<meta name="author" title="Ryosuke Niwa" href="mailto:rniwa@webkit.org">
+<meta name="assert" content="When the HTML parser creates an element for a token, it must perform a microtask checkpoint before invoking the constructor">
+<meta name="help" content="https://html.spec.whatwg.org/multipage/parsing.html#create-an-element-for-the-token">
+<meta name="help" content="https://html.spec.whatwg.org/multipage/webappapis.html#perform-a-microtask-checkpoint">
+<meta name="help" content="https://html.spec.whatwg.org/multipage/parsing.html#adoption-agency-algorithm">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/custom-elements-helpers.js"></script>
```

Comment:
mmh, it seems you ignored my comment https://github.com/web-platform-tests/wpt/pull/12447#discussion_r232938117 on the other PR. I thought "/resources/custom-elements-helper.js" gets the element at the root, so I think it should be "resources/custom-elements-helper.js" instead (without the initial /). Not sure why this is still working...

cc @Ms2ger

Reply:
@fred-wang  Just checked this  test is passing on webkit gtk

Reply:
OK, but can you please fix the links to custom-elements-helper.js here and the one added in https://github.com/web-platform-tests/wpt/pull/12447 ?

Reply:
OK, now I understand how the bad path was "fixed"... :-/

@darsh7807 Next time, please reply to questions we ask during review requests and be more explicit about how you addressed them. The move you silently did in https://github.com/web-platform-tests/wpt/pull/12447/commits/e342aed77e168325e86db3577c949c41b7f18f70 was not obvious from the commit message. Thank you.

Reply:
Hi, sorry for the delay in reply I had build issues on webkit, I have now corrected the file path now,